packages/polyfill/index.ts

```diff
@@ -21,10 +21,12 @@ import 'whatwg-fetch';
 // Promise
 import 'promise-polyfill/lib/polyfill';
 
-// ES6
+// ES Stable
 import 'core-js/features/array/find';
 import 'core-js/features/array/find-index';
 import 'core-js/features/object/assign';
+import 'core-js/features/object/entries';
+import 'core-js/features/object/values';
```

Comment:
If we keep using the Map in ChildChangeAccumulator, do we need to add `map/values` here?

Reply:
Added. :man_shrugging: